src/tests/ftest/daos_test/daos_core_test.py

```diff
@@ -24,17 +24,21 @@
 
 from daos_core_base import DaosCoreBase
 
+
 class DaosCoreTest(DaosCoreBase):
-    """
-    Runs just the non-rebuild daos_test tests
+    # pylint: disable=too-many-ancestors
+    """Runs the daos_test tests that do not require fault injection enabled.
 
     :avocado: recursive
     """
+
     def test_subtest(self):
-        """
-        Test ID: DAOS-1568
+        """Run daos_test tests/subtests.
```

Comment:
I don't think we should be dropping the `JIRA ID: DAOS-1568` part of the docstring?

Reply:
I fixed the issue because I had a typo in the pragma and the fault test was not skipped

Skip-fault-test: True   =>  Skip-fault-test: true